Makefile

```diff
@@ -32,15 +30,6 @@ validate-only:
 		echo "$$OUTPUT" ;\
 		exit 1 ;\
 	fi
-	@printf "=== $(INTEGRATION) === [ validate ]: running golint... "
-	@OUTPUT="$(shell golint $(SRC_DIR)...)" ;\
```

Comment:
why are we removing golint? if it is not needed anymore we can clean away as well "VALIDATE_DEPS    = golang.org/x/lint/golint" and `validate-deps:`

Reply:
`golint` is deprecated, should be replaced by golangci-lint.

Reply:
Yep, we have the newrelick infra checkers action that we should incorporate in a different PR.